### PR TITLE
NEXT-12691 - Add message to queue

### PR DESCRIPTION
**Pull request by Dominik28111**

### CODE REVIEW

File: guides/plugins/plugins/framework/message-queue/add-message-to-queue.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Add message to queue
+
+## Overview
+
+In this guide you'll learn how to create a message and add it to the queue.
+
+A [message](https://symfony.com/doc/current/messenger.html#creating-a-message-handler) is a simple PHP class that you want to dispatch over the MessageQueue.
```
Comment (lernhart) [inline guides/plugins/plugins/framework/message-queue/add-message-to-queue.md:7]: I'd prefer the terminology `object`.

```suggestion
A [message](https://symfony.com/doc/current/messenger.html#creating-a-message-handler) is a simple PHP object that you want to dispatch over the MessageQueue. 
```

File: guides/plugins/plugins/framework/message-queue/add-message-to-queue.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# Add message to queue
+
+## Overview
```
Comment (lernhart) [inline guides/plugins/plugins/framework/message-queue/add-message-to-queue.md:3]: I'm missing a general explanation, what a the message queue is. 

The message queue allows you to ... 

Maybe you can copy something from the symfony docs as some sort of introduction here

File: guides/plugins/plugins/framework/message-queue/add-message-to-queue.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    public function sendMessage(string $message): void
+    {
+        $this->bus->dispatch(new SmsNotification($message));
+    }
+}
+```
+{% endcode %}
+
+If we want to add metadata to our message, we can simply dispatch an `Symfony\Component\Messenger\Envelope` with the necessary
```
Comment (lernhart) [inline guides/plugins/plugins/framework/message-queue/add-message-to-queue.md:85]:
```suggestion
If we want to add metadata to our message, we can dispatch an `Symfony\Component\Messenger\Envelope` in our Service instead with the necessary 
```

File: guides/plugins/plugins/framework/message-queue/add-message-to-queue.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+{% code title="<plugin root>/src/Service/ExampleSender.php" %}
+```php
+public function sendMessage(string $message): void
+{
+    $message = new SmsNotification($message);
+    $this->bus->dispatch(
+        (new Envelope($message))
+        ->with(new DelayStamp(5000))
+    );
```
Comment (lernhart) [inline guides/plugins/plugins/framework/message-queue/add-message-to-queue.md:98]:
```suggestion
    $this->bus->dispatch(
        (new Envelope($message))
            ->with(new DelayStamp(5000))
    );
```

**Review by lernhart — CHANGES_REQUESTED** [review]

**Review by lernhart — APPROVED** [review]

**Review by PaddyS — CHANGES_REQUESTED** [review]
Dead links

**Pull request merged**